<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-SRCJA9U0" failures="1" tests="5" name="Test" time="196.205" errors="0" timestamp="2025-02-15T20:18:56 IST">
  <testcase classname="testcases.LoginTestCase" name="successfulLogin" time="9.417"/>
  <testcase classname="testcases.LoginTestCase" name="loginWithInvalidPassword" time="14.29">
    <skipped/>
  </testcase> <!-- loginWithInvalidPassword -->
  <testcase classname="testcases.LoginTestCase" name="loginWithInvalidPassword" time="29.891">
    <skipped/>
  </testcase> <!-- loginWithInvalidPassword -->
  <testcase classname="testcases.LoginTestCase" name="loginWithInvalidPassword" time="25.668">
    <skipped/>
  </testcase> <!-- loginWithInvalidPassword -->
  <testcase classname="testcases.LoginTestCase" name="loginWithInvalidPassword" time="7.62">
    <failure type="java.lang.AssertionError" message="Alert text did not match expected [Invalid credentialst] but found [Invalid credentials]">
      <![CDATA[java.lang.AssertionError: Alert text did not match expected [Invalid credentialst] but found [Invalid credentials]
at testcases.LoginTestCase.loginWithInvalidPassword(LoginTestCase.java:50)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- loginWithInvalidPassword -->
</testsuite> <!-- Test -->
